src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -30,7 +30,7 @@ static ucs_config_field_t uct_cuda_ipc_iface_config_table[] = {
      "Max number of event completions to pick during cuda events polling",
       ucs_offsetof(uct_cuda_ipc_iface_config_t, max_poll), UCS_CONFIG_TYPE_UINT},
 
-    {"MAX_STREAMS", "16",
+    {"MAX_STREAMS", "128",
```

Comment:
can we use UCS_PP_MAKE_STRING(UCT_CUDA_IPC_MAX_PEERS) here?
what happens if the user sets MAX_STREAMS config to a value larger than UCT_CUDA_IPC_MAX_PEERS?

Reply:
Having larger MAX_STREAMS than UCT_CUDA_IPC_MAX_PEERS allows cuda-ipc transport to round-robin select among more number of streams. It wouldn't be a functional or performance issue in my opinion although it is unlikely to improve performance.